Automattic/newspack-plugin
# Deprecate Stripe Reader Revenue separate platform

### All Submissions:






### Changes proposed in this Pull Request:



Unifies on-site Reader Revenue platforms, so that "Newspack" is the only one. 

This is a breaking change. Recurring donations will no longer result in Stripe Subscriptions – instead, WC Subscriptions will be created. If Stripe was set as the Reader Revenue platform, it will be changed to Newspack.  

If Stripe is set as the RR platform on a site, with these changes there will be consequences:

1. New recurring donations will result in WC Subscriptions instead of native Stripe Subscriptions
2. Existing Stripe Subscriptions will no longer be synchronised as orders to WC

### How to test the changes in this Pull Request:

1. On `master`, configure Stripe as the Reader Revenue platform
3. Switch to this branch
4. Observe the Reader Revenue platform in the Reader Revenue wizard is now set to "Newspack"
6. Make a recurring donation and then a one-time donation through the Donate block
7. Observe both processed in Stripe (in [Stripe Dashboard](https://dashboard.stripe.com/dashboard)) as single transactions 
8. Observe the recurring donation has resulted in a WC Subscription, and the one-time donation in a single order

### Other information:
